doc/model.rst

```diff
@@ -553,16 +596,18 @@ You can change this dictionary directly, or with the :meth:`Model.set_param_hint
 method. Either way, these parameter hints are used by :meth:`Model.make_params`
 when making parameters.
 
-An important feature of parameter hints is that you can force the creation
-of new parameters with parameter hints. This can be useful to make derived
-parameters with constraint expressions. For example to get the full-width
-at half maximum of a Gaussian model, one could use a parameter hint of:
+Parameter hints also allow you to create new parameters. This can be useful to
+make derived parameters with constraint expressions. For example to get the
+full-width at half maximum of a Gaussian model, one could use a parameter hint
+of:
 
 .. jupyter-execute::
 
     mod = Model(gaussian)
     mod.set_param_hint('fwhm', expr='2.3548*sigma')
 
+With that definition, the value (and uncertainty) of the ``fwhm`` parameter
+will be reported in the output of any fit done with that model.
 
```

Comment:
I think it could be good to add an explicit example of the `make_params` with kwargs, like `make_params(a=1.0, b=0.3, min=0, max=1.0)`.